src/server/backend.py

```diff
@@ -37,7 +37,9 @@ def __init__(self, *args, memory_cache: MemoryCache, backend_dtype: Optional[tor
         self.backward_pool = PrioritizedTaskPool(
             self.backward, max_batch_size=max_batch_size, name=f"{self.name}_backward"
         )
-        self.dtype = backend_dtype if backend_dtype else self.module.input_layernorm.weight.dtype
```

Comment:
The default backend dtype is now chosen in `ModuleContainer.create()`, since it is also needed for the schemas.